nimbus-ui/nimbusui/src/app/components/platform/validators/custom.validators.ts

```diff
@@ -74,5 +74,13 @@ export class CustomValidators {
         };
     }
 
+    static past(control: FormControl) {
+        return null;
+    }
```

Comment:
I'm not sure if returning null here is the best option.  @sandeepmantha thoughts?